tap_github/streams.py

```diff
@@ -906,6 +919,32 @@ def http_headers(self) -> dict:
     ).to_dict()
 
 
+class PullRequestFilesStream(GitHubStream):
+    """Defines 'PullRequestFiles' stream."""
+
+    name = "pull_request_files"
+    path = "/repos/{org}/{repo}/pulls/{pull_number}/files"
+    primary_keys = ["filename"]
```

Comment:
I think this will cause problems, as filenames are most likely not going to be globally unique. This [comment](https://github.com/MeltanoLabs/tap-github/pull/13#discussion_r707816224) might be helpful.

Reply:
Out of the output: 
![image](https://user-images.githubusercontent.com/25571542/141050760-bfb7885f-a342-425a-b750-284e50772834.png)
I'm not sure what else could be the key. Any ideas? Since hashes could be the same (unlikely) I chose filename, since it looks like it gives the full filename. What do you think?